src/util/DefaultValueSizeGetter.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2023, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Andre Schlegel (August of 2023,
+// schlegea@informatik.uni-freiburg.de)
+#pragma once
+
+#include <string>
+
+#include "util/Cache.h"
+#include "util/MemorySize/MemorySize.h"
+
+namespace ad_utility {
+// Multiple default `ValueSizeGetter` for usage in caches. For information, what
+// a `ValueSizeGetter` is, please see `Cache.h`.
+
+// `ValueSizeGetter` for caches, which simply calls the `sizeof` operator.
+struct SizeOfSizeGetter {
+  ad_utility::MemorySize operator()(const auto& obj) const {
```

Comment:
maybe add a `requires std::is_trivially_copyable_v` in here, to prevent accidental use (maybe there is even a concept for this type trait).

Reply:
I don't follow. We never copy `obj`, so why would we need that requirement?

Reply:
Done, see [ff0d6de](https://github.com/ad-freiburg/qlever/pull/1111/commits/ff0d6dea208918b1abc450a20a003140c7be3449).